benchbuild/cli/project.py

```diff
@@ -33,6 +35,40 @@ def main(self, *projects: str) -> int:
         return 0
 
 
+@BBProject.subcommand("details")
+class BBProjectDetails(cli.Application):
+    """Show details for a project."""
+
+    def main(self, project: str) -> int:
+        index = bb.populate([project], [])
+        if not index.values():
+            print(f'Project named {project} not found in the registry.')
+            print('Maybe it is not configured to be loaded.')
+            return -1
+        for project_cls in index.values():
+            print_project(project_cls)
+        return 0
+
+
+def print_project(project: tp.Type[Project]) -> None:
+    print(f'project: {project.NAME}')
+    print('source:')
+    for source in project.SOURCE:
+        print(' ', f'{source.remote}')
+        print('versions')
+        for v in source.versions():
+            print(' ', v)
+    containers = []
+    if isinstance(project.CONTAINER, ContainerImage):
+        containers.append(project.CONTAINER)
+    else:
+        containers.extend(containers)
+
+    print('container:')
+    for container in containers:
+        print(' ', container)
```

Comment:
Does this print nicely when `CONTAINER` is a list of `(range, image)` tuples (i.e., the `else` case above)?
As far as I know, python uses `__repr__` to print tuple members, otherwise #378 would help.

Reply:
Should print nicely with #378